js/functions.js

```diff
@@ -0,0 +1,67 @@
+/* Первая функция
+Javascript должен проверять длинну строки, сравнивать её с заданным значением
+и возвращать true если условие выполняется и false если нет
+*/
+
+const checkStringLength = (string, length) => string.length <= length;
+
+checkStringLength('Шла Саша по шоссе', 25);
+
+
+/*Вторая функция
+JS должен проверять совпадают ли символы при прямом и обратном прочтение строки,
+если совпадают возвращать true, иначе false*/
+
+const isWordPalindrome = (string) => {
+  for (let i = 1; i <= string.length; i++) {
+    string = string.toLowerCase();
+    if (string[i - 1] === (string[String.length - i])) {
+      continue;
+    }
+    return false;
+  }
+  return true;
+};
+
+isWordPalindrome('ТопОт');
+
+
+//Вторая функция с усложнением на пробелы
+
+const isPhrasePolindrom = (deleteSpacePhrase) => {
+  deleteSpacePhrase = deleteSpacePhrase
+    .toLowerCase()
+    .replaceAll(' ','');
+  const ReversePhase = deleteSpacePhrase
+    .split('')
+    .reverse()
+    .join('');
+  return ReversePhase === deleteSpacePhrase;
+};
+
+isPhrasePolindrom('Лёша на полке клопа нашёл ');
+
+//Третья функция с извлечением числа из строки
+const extractNumber = (string) => {
```

Comment:
```suggestion
const extractNumbers = (string) => {
```
А вообще я бы заменил на parseNumbers так как именно это мы делаем